openage/convert/slp.pyx

```diff
@@ -405,12 +526,12 @@ cdef class SLPFrame:
                 dpos += 1
                 color = self.get_byte_at(dpos)
 
-                # TODO: verify this. might be incorrect.
-                # color = ((color & 0b11001100) | 0b00110011)
-
-                # SpecialColor class preserves the calculation of
-                # player*16 + color
                 for _ in range(cpack.count):
+                    # TODO: verify this. might be incorrect.
+                    # color = ((color & 0b11001100) | 0b00110011)
+
+                    # SpecialColor class preserves the calculation of
+                    # player*16 + color
```

Comment:
i think i removed the `SpecialColor` class some time ago and replaced it with the enum that contains `color_player`. The comment should therefore read "the `color_player` enum later resolves the special case for the player color".

Reply:
> i think i removed the SpecialColor class some time ago

It's still in `slp.pyx` (Line 24). I guess it should be removed then?

Reply:
Also, I removed the comment entirely because the player color is calculated on the fly in `determine_rgba_matrix(..)`